src/js/components/search/SearchSidebar.jsx

```diff
@@ -118,8 +172,18 @@ const defaultProps = {
 
 export default class SearchSidebar extends React.Component {
     render() {
+        const indexOfUnreleased = filters.options.findIndex((option) => (
```

Comment:
Instead of doing this and then reducing the array, we could just have a method or code that just says, `const releasedFilters = GlobalConstants.CARES_ACT_RELEASED ? filters.options.splice(index, 0, DEFOBJECT) : filters` or some simple if statement to the fact, since this DEF thing is a one off.